Scripts/Source/minai_Arousal.psc

```diff
@@ -175,11 +213,34 @@ function WriteClothingString(actor akActor, actor player, bool isYou=false, acto
 		if (currentActor != None)
 			String actorName = main.GetActorName(currentActor)
 			Armor cuirass = currentActor.GetWornForm(cuirassSlot) as Armor
-			if !cuirass
+			if cuirass == None
 				main.RegisterAction(actorName + " is naked.")
 			else
 				main.RegisterAction(actorName + " is wearing " + cuirass.GetName())
 			EndIf
+			if bHasTNG
+				if currentActor.GetActorBase().GetSex() == 0 && IsTNGExposed(currentActor) || currentActor.HasKeyword(TNG_Gentlewoman) && IsTNGExposed(currentActor)
```

Comment:
Instead of calling IsTNGExposed 3 times, call it once, and store the result in a variable.